src/renderer/store/exportIo/actions.ts

```diff
@@ -4,9 +4,15 @@ export const START_EXPORT = 'START_EXPORT';
 export const EXPORT_PROGRESS_UPDATE = 'EXPORT_PROGRESS_UPDATE';
 export const FINISH_EXPORT = 'FINISH_UPDATE';
 
-export const startExport: () => Action<null> = () => ({
+export const startExport: (
```

Comment:
Looks like the redux isn't getting updated. Is this action getting dispatched at all?

Reply:
ah fk I think I messed up while rebasing. Fixing now

Reply:
done. getting dispatched from "initiate-export-project" [here](https://github.com/patrickbrett/mlvet/pull/56/files#:~:text=store.dispatch(startExport(currentProject.id%2C%20filePath))%3B)